sample/001_banner/app.yaml

```diff
@@ -6,6 +6,13 @@ status_response: "ok" # default response string for /status endpoint
 
 gdpr:
   default_value: "0" # disable gdpr, explicitly specifying a default value is a requirement in prebid server config
+  vendorlist_fetcher:
+    max_concurrency_init_fetch_latest_version: 2
+    max_concurrency_init_fetch_specific_version: 20
+  timeouts_ms:
+    init_vendorlist_fetches: 30000
+    active_vendorlist_fetch: 30000
```

Comment:
if ` init_vendorlist_fetches` is 0 - then you literally set 0 sec as the timeout of preloading the vendor list, what you end up seeing the preload will have context deadline exceeded/timeout

Reply:
Hi @zhongshixi 

I am not asking what are those new configuration keys for.

I am asking whether or not the Prebid Server software works if you don't provide them.

Reply:
@sebhtml i am not following

1. even without my change  the issue `context deadline issue` still happen, because you can not really set 0 timeout , the default setting is using `0` as timeout
```
v.SetDefault("gdpr.timeouts_ms.init_vendorlist_fetches", 0)
v.SetDefault("gdpr.timeouts_ms.active_vendorlist_fetch", 0)
```

2. the default is using sequential execution, the default setting added in this PR is also sequential execution
```
v.SetDefault("gdpr.vendorlist_fetcher.max_concurrency_init_fetch_latest_version", 1)   // by default one fetch at a time
	v.SetDefault("gdpr.vendorlist_fetcher.max_concurrency_init_fetch_specific_version", 1) // by default one fetch at a time
```

maybe i am wrong, could you point out exactly where it is not backward-compatible?

Reply:
I don't know how to rephrase it. So I let it go.